guide/english/computer-science/threads/index.md

```diff
@@ -32,6 +32,35 @@ if __name__ == "__main__":
     for i in range(1, 11):
         threading.Thread(target=print_number, args=(i,)).start()
 ````
+## Java Implementation
+````java
+class print_thread implements Runnable
+{
+    int n=0;
+    public print_thread(int n) {
+        this.n=n;
+    }
+    
+    @Override
+    public void run() {
+        
+            System.out.println(n+" <-- From Thread number "+n);
+    }
+    
+}
+public class MainClass {
+    
+    public static void main(String[] args) {
+        for(int i=1;i<11;i++)
+        {
+            new Thread(new print_thread(i)).start();
```

Comment:
Please add the expected output (print statements) from this line, after the code sample (line 64).